flash/image/classification/integrations/baal/__init__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import flash
+from flash.core.utilities.imports import requires
+from flash.image.classification.integrations.baal.loop import ActiveLearningDataModule, ActiveLearningLoop  # noqa F401
+
+
+class ActiveLearningTrainer(flash.Trainer):
+    @requires("baal")
+    def __init__(self, *args, label_epoch_frequency: int = 1, inference_iteration: int = 2, **kwags):
```

Comment:
should probably not support positional args :)

Reply:
Yes, I don't think the Trainer as any anyway :)

Reply:
Removed !